### ISSUE TITLE
Getting this bug on new update version: 47.1.3

### ISSUE THREAD

**Issue by zifahm** [pinterest/gestalt#2034]
Platform: Web

1.Use a Text Field

Error: TextField props accessibilityHidePasswordLabel, accessibilityShowPasswordLabel are missing translations — please add translations to I18nProvider

![Screenshot from 2022-04-01 12-00-12](https://user-images.githubusercontent.com/3761062/161207813-16bbfc48-680b-4cad-8248-738182c2e781.png)

Workaround: go back to version 46.7.12
